Objective-C/Internal/Replicator/CBLTrustCheck.mm

```diff
@@ -201,6 +201,7 @@ - (NSURLCredential*) acceptOnlySelfSignedCert: (NSError**)outError {
     if (!isSelfSigned) {
         MYReturnError(outError, NSURLErrorServerCertificateUntrusted, NSURLErrorDomain,
             @"Server SSL certificate is not self-signed");
+        CBLLogVerbose(WebSocket, @"Server SSL certificate is not self-signed, force trusting...");
```

Comment:
I think there is a bug here that it should return nil instead of doing force trust.